README.md

```diff
@@ -1 +1,38 @@
-# webpack-Demo
\ No newline at end of file
+# Todo list app
+
+> A Javascript to do list web app that keep track of activities.
+
+## Built With
+
+- Major languages: HTML CSS & JavaScript
+- Frameworks: none
+- Technologies used: Webpack
+
+[Webpage Link ](https://collinsmezie.github.io/To-Do-List/) 
+
```

Comment:
- [ ]  Your README looks great 👍🏻  but I think it will be great to add instructions to set the project up locally and instructions on how to run the project. This includes `npm install` , `npm start` etc ,  and a screenshot of your project. please refer to this [template](https://github.com/microverseinc/readme-template), you can copy and paste the content but remember to customize it to your project.